cs-config/cs_config/functions.py

```diff
@@ -37,12 +48,15 @@ def get_inputs(meta_params_dict):
     metaparams.adjust(meta_params_dict)
 
     params = CruncherParams()
-    policy_params = TCParams()
+    policy_params = Policy()
 
     policy_params.set_state(
         year=metaparams.year.tolist())
 
     filtered_pol_params = OrderedDict()
+    policy_params._schema["operators"].update(
+        {"label_to_extend": None, "uses_extend_func": False, "array_first": False}
+    )
```

Comment:
@Peter-Metz I'll go back and fix this once https://github.com/PSLmodels/ParamTools/issues/117 is resolved.